docs/UserGuide.md

```diff
@@ -359,6 +366,15 @@ Examples:
 
 - `toggle` changes the application theme.
 
+### Navigating Command History : 
+
+Easily cycle through your previous **successfully executed** commands by using the Up and Down arrow keys on your keyboard.
+
+Format: `Up arrow`
```

Comment:
Maybe specify on the keyboard and not in the command box later some people just type `up arrow`, then they see invalid command.